API_DOCUMENTATION.md

```diff
@@ -9,5 +9,48 @@
   **Response:**
   {body: string}
 
+  ---
+
+#### POST "/messages/"
+  *Create a new message with given JSON body information*
+  
+  **Params:**
+  {
+  text: string,
+  senderId: int,
+  receiverId: int
+  }
+
+  **Response:**
+  STATUS 200 OK
+
+#### GET "/messages/conversations/{userId}"
+  *Get ids of the users the userId identified user is having conversations with*
+  
+  **Params:**
+  userId: int
+
+  **Response:**
+  [id: int]
+
+#### GET "/messages/conversation/{userId1}/{userId2}"
```

Comment:
Remove the messages prefix. The endpoint will be GET "/conversations/{userId1}/{userId2}". We have a collection of conversations and we query a specific one by an unique identifier - combination of two user ids.

Reply:
Tehnically we could also say GET /messages/{user1}/{user2} since the query returns all messages between 2 users (there isnt a seperate conversation model for us)